base/gfspclient/metadata.go

```diff
@@ -404,3 +404,49 @@ func (s *GfSpClient) GetGroupList(
 	}
 	return resp.Groups, resp.Count, nil
 }
+
+func (s *GfSpClient) ListBucketsByBucketID(
+	ctx context.Context,
+	bucketIDs []int64,
+	includeRemoved bool,
+	opts ...grpc.DialOption) ([]*types.Bucket, error) {
```

Comment:
These function params can be moved to one line, don't exceed 120 columns in one line.

Reply:
fixed